workflow/environments/org/nesi/create_env_maui.sh

```diff
@@ -54,6 +54,8 @@ xargs -n 1 -a ${env_path}/workflow/workflow/environments/org/nesi/maui_python3_r
 source "${env_path}/workflow/workflow/environments/org/nesi/create_env_common_post.sh"
 
 cd ${env_path}/qcore
-python setup.py install
-cd ${env_path}/workflow
-python setup.py install
+pip install -r requirements.txt
```

Comment:
If there are extra pip requirements would it be better to add them to the maui/mahuika_python3_requirements.txt files above?